pressure.go

```diff
@@ -0,0 +1,66 @@
+package unit
+
+import (
+	"encoding"
+	"fmt"
+)
+
+// Pressure is the force applied perpendicular to the surface of an object per unit area over which that
+// force is distributed.
+type Pressure float64
+
+var _ encoding.TextUnmarshaler = (*Pressure)(nil)
+
+// Pascal is the SI unit for measuring Pressure.
+const Pascal Pressure = 1
+
+const pascalSymbol = "Pa"
+
+// KiloPascal is 1000 times the SI unit Pascal.
+const KiloPascal Pressure = 1000 * Pascal
+
+const kiloPascalSymbol = "kPa"
+
+// KiloPascal returns p with the unit of kPa.
+func (p Pressure) KiloPascal() float64 {
+	return p.Get(KiloPascal)
+}
+
+// Bar is 100000 times the SI unit Pascal.
+const Bar Pressure = 1e6 * Pascal
```

Comment:
The comment is correct, but the code is not. This should be `1e5`.

Reply:
Very good catch, thanks!